wrapper.sh

```diff
@@ -1,120 +1,114 @@
 #!/bin/sh
 
-if [ "x$DEBUG" != "x" ]
-then
+if [ "x$DEBUG" != "x" ]; then
```

Comment:
Updated formatting for all if statements, to improve readability (imo). Can revert if preferred.